public/modules/custom/infofinland_municipality_selection/src/Form/InfofinlandMunicipalitySelectionBlockForm.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\infofinland_municipality_selection\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Infofinland Municipality Selection block form
+ */
+class InfofinlandMunicipalitySelectionBlockForm extends FormBase {
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'infofinland_municipality_selection_block_form';
+  }
+
+  /**
+   * @param array $form
+   * @param FormStateInterface $form_state
+   * @return array
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $items = [];
+
+    // Get the term storage.
+    $entity_storage = \Drupal::entityTypeManager()->getStorage('taxonomy_term');
```

Comment:
Dependency injections should be used in classes instead of static calls. Example: https://www.drupal.org/docs/drupal-apis/services-and-dependency-injection/dependency-injection-for-a-form